<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SEEG_masking</class>
 <widget class="qMRMLWidget" name="SEEG_masking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>789</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>3. Enhancement ctp3D</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_ctp">
        <property name="text">
         <string>Input Volume: (ctp3D)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_ctp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_ROI_norm">
        <property name="text">
         <string>Input Volume: (ROI_norm)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_ROI_norm">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_ctp">
        <property name="text">
         <string>Enhance Image </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Input enhanced volume:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_filter">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Apply more filters if you want</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ctkCheckableComboBox" name="Checkable_filters">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Morphological Operations</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Canny Edge Detection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High Pass Sharpening</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_filters">
        <property name="text">
         <string>Apply filters </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>1. Mask Creation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input volume (if you have norm/brain scan): </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector">
        <property name="toolTip">
         <string>Pick the input to the algorithm.</string>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true">
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>inputVolume</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="applyButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Run the algorithm.</string>
        </property>
        <property name="text">
         <string>âœ… Apply Masking Norm (the electrodes are gonna be here)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save mask</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
     <property name="text">
      <string>2. Mask creation if your scan has mores tissues apart from the brain ðŸ’€</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSelector_saveMaskAI">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Input Volume: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="newButton_save">
        <property name="text">
         <string>Create Mask and save it :D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkCheckableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SEEG_masking</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
